execution/setContext.sh

```diff
@@ -70,16 +70,17 @@ if [[ (-f "environment.json") ]]; then
     cd ../../../config
 fi
 
-if [[ -f "account.json" ]]; then
+if [[ -f "${current_dir}/account.json" ]]; then
```

Comment:
This won't work with the previous `cd` operations.

Reply:
That was the problem. These are trying to determine the location that you are currently in. 
The solutions_dir checking above ends up putting the cwd at the point of execution in the infrastructure dir the current location instead of the config dir to see if this file exists. 

So using the current_dir for the ones that don't rely on nesting like segment and environment does seemed like a better approach.

Reply:
@ml019 Any further advise on this? I think the issue is in the ancestor lookup which works ok for segment solutions in the new or old CMDB structure but doesn't appear to work for accounts that don't have a solutions dir to match from.